docs/docusaurus.config.ts

```diff
@@ -47,6 +47,11 @@ const config: Config = {
         src: "img/pocket-id.png",
       },
       items: [
+        {
+          to: "#version", // Use a hash link to trigger the onClick
+          label: "Version",
```

Comment:
I would leave the label empty while the version is loading.
```suggestion
          label: " ",
```